src/cfml/system/config/server.schema.json

```diff
@@ -261,97 +323,180 @@
 							"description": "HTTP port to use",
 							"type": "number",
 							"default": 0
 						}
 					}
 				},
+				"HTTP2": {
+					"title": "HTTP2 Settings",
+					"description": "Configure HTTP2",
+					"type": "object",
+					"properties": {
+						"enable": {
+							"title": "HTTP2 Enable",
+							"description": "Enable HTTP2 for this serer",
+							"type": "boolean",
+							"default": true
+						}
+					}
+				},
 				"SSL": {
 					"title": "SSL",
 					"description": "Configure the HTTPS listener on the server",
 					"type": "object",
 					"properties": {
 						"enable": {
-							"title": "Enable",
+							"title": "SSL Enable",
 							"description": "Enable HTTPS for this server",
 							"type": "boolean",
 							"default": false
 						},
 						"port": {
-							"title": "Port",
+							"title": "SSL Port",
 							"description": "HTTPS port to use",
 							"type": "number",
 							"default": 1443
 						},
 						"certFile": {
-							"title": "Cert File",
+							"title": "SSL Cert File",
 							"description": "Path to SSL cert file",
 							"type": "string",
 							"default": ""
 						},
 						"keyFile": {
-							"title": "Key File",
+							"title": "SSL Key File",
 							"description": "Path to SSL key file",
 							"type": "string",
 							"default": ""
 						},
 						"keyPass": {
-							"title": "Key Pass",
+							"title": "SSL Key Pass",
 							"description": "Password for SSL key file",
 							"type": "string",
 							"default": ""
+						},
+						"forceSSLRedirect": {
+							"title": "Force SSL Redirect",
+							"description": "Whether to redirect all HTTP traffic over to HTTPS using a 301 status code",
+							"type": "boolean",
+							"default": false
+						},
+						"HSTS": {
+							"title": "HSTS",
+							"description": "HTTP Strict Transport Security configuration",
+							"type": "object",
+							"properties": {
+								"enable": {
+									"title": "HSTS Enable",
+									"description": "Whether to add a Strict-Transport-Security HTTP header",
+									"type": "boolean",
+									"default": false
+								},
+								"maxAge": {
+									"title": "HSTS Max Age",
+									"description": "How many seconds to remember to use HTTPS",
+									"type": "number",
+									"default": 31536000
+								},
+								"includeSubDomains": {
+									"title": "HSTS Include Subdomains",
+									"description": "Whether the HSTS header applies to all subdomains",
+									"type": "boolean",
+									"default": false
+								}
+							},
+							"required": [ "enable" ]
+						},
+						"clientCert": {
+							"title": "SSL Client Cert",
+							"description": "",
+							"type": "object",
+							"properties": {
+								"mode": {
+									"title": "Client Cert Mode",
+									"description": "",
+									"type": "string",
+									"default": ""
+								},
+								"CACertFiles": {
+									"title": "CA Cert Files",
+									"description": "",
+									"type": "string",
```

Comment:
This can be a string or an array

Reply:
Fixed. FYI, I couldn't find any documentation on `web.SSL.clientCert`.